package.json

```diff
@@ -49,57 +49,53 @@
-    "@storybook/react": "^6.3.0-alpha.14",
+    "@babel/core": "^7.16.7",
+    "@storybook/builder-webpack5": "^6.3.12",
+    "@storybook/manager-webpack5": "^6.3.12",
+    "@storybook/react": "^6.3.12",
     "@types/draft-js": "^0.10.45",
-    "@types/react": "^16.9.53",
-    "@types/react-redux": "^7.1.16",
-    "@types/redux": "^3.6.0",
-    "@typescript-eslint/eslint-plugin": "^4.5.0",
-    "@typescript-eslint/parser": "^4.5.0",
-    "@wagtail/stylelint-config-wagtail": "^0.2",
-    "autoprefixer": "^9.8.0",
-    "babel-eslint": "^10.1.0",
-    "babel-loader": "^8.2.2",
-    "cssnano": "^4.1.10",
-    "dotenv-webpack": "^7.0.2",
-    "enzyme": "^3.9.0",
-    "enzyme-adapter-react-16": "^1.9.1",
-    "enzyme-to-json": "^3.3.0",
-    "eslint": "^7.20.0",
-    "eslint-config-wagtail": "^0.1.1",
-    "eslint-import-resolver-webpack": "^0.8.1",
-    "eslint-plugin-import": "^1.8.1",
-    "eslint-plugin-jsx-a11y": "^1.5.3",
-    "eslint-plugin-react": "^5.2.2",
-    "expose-loader": "^1.0.3",
-    "gulp": "^4.0.2",
-    "gulp-dart-sass": "^1.0.2",
-    "gulp-postcss": "^8.0.0",
-    "gulp-rename": "^1.2.2",
-    "gulp-size": "^3.0.0",
-    "gulp-sourcemaps": "~2.6.1",
-    "gulp-util": "~3.0.8",
-    "html-webpack-plugin": "^5.3.1",
+    "@types/jest": "^26.0.24",
+    "@types/react": "^16.14.21",
+    "@types/react-redux": "^7.1.20",
+    "@typescript-eslint/eslint-plugin": "^5.8.0",
+    "@typescript-eslint/parser": "^5.8.0",
+    "@wagtail/eslint-config-wagtail": "^0.4.0",
+    "@wagtail/stylelint-config-wagtail": "^0.3.2",
+    "autoprefixer": "^10.4.0",
+    "babel-loader": "^8.2.3",
+    "copy-webpack-plugin": "^10.2.0",
+    "css-loader": "^6.5.1",
+    "cssnano": "^5.0.14",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.6",
+    "enzyme-to-json": "^3.6.2",
+    "eslint": "^8.5.0",
+    "eslint-config-airbnb": "^19.0.2",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-import": "^2.25.3",
+    "eslint-plugin-jsx-a11y": "^6.5.1",
+    "eslint-plugin-react": "^7.28.0",
+    "eslint-plugin-react-hooks": "^4.3.0",
+    "expose-loader": "^3.1.0",
     "jest": "^26.6.3",
-    "npm-run-all": "^4.1.5",
-    "postcss-custom-properties": "^10.0.0",
-    "react-axe": "^3.1.0",
-    "react-test-renderer": "^16.13.1",
+    "mini-css-extract-plugin": "^2.4.5",
+    "postcss": "^8.4.5",
+    "postcss-loader": "^6.2.1",
+    "react-axe": "^3.5.4",
+    "react-test-renderer": "^16.14.0",
     "redux-mock-store": "^1.3.0",
-    "stylelint": "^13.13.1",
-    "ts-jest": "^26.4.1",
-    "ts-loader": "^9.1.1",
-    "typescript": "^4.0.3",
-    "webpack": "^5.35.1",
-    "webpack-cli": "^3.3.12"
+    "sass": "^1.45.1",
+    "sass-loader": "^12.4.0",
+    "stylelint": "^14.2.0",
+    "ts-jest": "^26.5.6",
+    "ts-loader": "^9.2.6",
+    "typescript": "^4.5.4",
+    "webpack": "^5.65.0",
+    "webpack-cli": "^4.9.1"
   },
   "dependencies": {
     "core-js": "^2.5.3",
+    "dateformat": "^2.2.0",
```

Comment:
This is a dependency that was there already (used to format dates in comments only), it was just not declared at the top level. It’s completely inappropriate for Wagtail, as the file size is enormous for the value we’re getting from this. I would like us to remove this in a follow-up issue.

Reply:
Can an issue be raised from this?
Additional to that we may want to add a future ticket to limit the build filesize for things like this to get flagged early.

Reply:
Done: #https://github.com/wagtail/wagtail/issues/7904. This will be enforced by [no-extraneous-dependencies](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-extraneous-dependencies.md) once we have our ESLint configuration upgrade merged :)